packages/lesswrong/server/collections/ultraFeedEvents/mutations.ts

```diff
@@ -35,9 +46,45 @@ export async function createUltraFeedEvent({ data }: CreateUltraFeedEventInput,
   return document;
 }
 
+export async function updateUltraFeedEvent(args: { selector: SelectorInput, data: CreateUltraFeedEventDataInput }, context: ResolverContext) {
+  const { selector, data: inputData } = args;
```

Comment:
We should definitely validate the `event` in the update data against a zod schema that constrains it to whatever's allowed for feedback events (or other events, if we want to permit updates for those to go through).